From ff3dd55d0a237c5e6ce51bfa04dea831843f1af5 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 5 Nov 2012 16:55:07 +0800
Subject: [PATCH 094/310] ENGR00232414 Bootanimation be started 10s after
 console started

Surfacefinger will guarantee caller the permission if its pid is AID_GRAPHICS.
For those caller's pid not AID_GRAPHICS, Surfaceflinger will count on
ServiceManager's permission service to check the permission. ServiceManager
will keep waiting for a valid permission service registered. Permission
Service will be registered in ActivityManager's creation, which is around
10s after console started.

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx6/etc/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index 85acea9..9bf386e 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -486,7 +486,7 @@ service media /system/bin/mediaserver
 
 service bootanim /system/bin/bootanimation
     class main
-    user system
+    user graphics
     group graphics
     disabled
     oneshot
-- 
1.8.0

